language_container/exaslct_scripts/exaslct.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+
+EXASLCT_GIT_REF="0.9.0"
```

Comment:
We are already at version 0.14.0. Please have a look, how to update. https://github.com/exasol/script-languages-container-tool#starter-scripts. I suggest you add the container-tool package to the dev dependencies.